openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml

```diff
@@ -282,12 +291,16 @@
 - control_key: IA-4 (5)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: complete
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-324
+        OCP offloads (via: https://access.redhat.com/documentation/en-us/
+        openshift_container_platform/4.3/html/authentication/
+        understanding-identity-provider)
+        responsibility for authentication, session management, and OAuth
+        controls to the hosting environment’s LDAP and leverages CAC/PKI,
+        FIPS 140-2 compliant ciphers, SSO and does not provide for group
+        accounts.
```

Comment:
Example of this could be the "login with GitHub" or "login with Google" buttons on websites. Can OpenShift be configured in such a manner (eg "login with DoD PKI!") and magically accept dynamic account creation?